pkg/handlers/app.go

```diff
@@ -383,3 +389,35 @@ func validateCustomAppMetricsData(data CustomAppMetricsData) error {
 
 	return nil
 }
+
+func SendAppInstanceTags(w http.ResponseWriter, r *http.Request) {
+	request := SendAppInstanceTagsRequest{}
+	if err := json.NewDecoder(r.Body).Decode(&request); err != nil {
+		t, ok := err.(*json.UnmarshalTypeError)
+		if ok {
+			logger.Debugf("failed to decode instance-tag request: %s value is not a string", t.Field)
```

Comment:
think it is worth having this log at the Error level so that we would see it in a support bundle?